Constitution.txt

```diff
@@ -1,490 +1,264 @@
+
+2: Each State shall appoint, in such Manner as the Legislature thereof may direct, a Number of Electors, equal to the whole Number of Senators and Representatives to which the State may be entitled in the Congress: but no Senator or Representative, or Person holding an Office of Trust or Profit under these United States, shall be appointed an Elector.
+
+3: The District constituting the seat of government of these United States shall appoint in such manner as the Congress may direct: A number of electors of President and Vice President equal to the whole number of Senators and Representatives in Congress to which the District would be entitled if it were a state, but in no event more than the least populous state; they shall be in addition to those appointed by the states, but they shall be considered, for the purposes of the election of President and Vice President, to be electors appointed by a state; and they shall meet in the District and perform the same duties as the Electors of the States.
 
-2: Each State shall appoint, in such Manner as the Legislature thereof may direct, a Number of Electors, equal to the whole Number of Senators and Representatives to which the State may be entitled in the Congress: but no Senator or Representative, or Person holding an Office of Trust or Profit under the United States, shall be appointed an Elector.
+4: The Electors shall meet in their respective States, and vote by Ballot for one Person who shall not be an Inhabitant of the same State with themselves. And they shall make a List of all the Persons voted for, and of the Number of Votes for each; which List they shall sign and certify, and transmit sealed to the Seat of the Government of these United States, directed to the President of the Senate. The President of the Senate shall, in the Presence of the Senate and House of Representatives, open all the Certificates, and the Votes shall then be counted. The Person having the greatest Number of Votes shall be the President, if such Number be a Majority of the whole Number of Electors appointed; and if no Person have a Majority, then from the five highest on the List the House of Representatives shall immediately choose by Ballot one of them for President. But in choosing the President, the Votes shall be taken by States, the Representation from each State having one Vote; A quorum for this Purpose shall consist of a Member or Members from two thirds of the States, and a Majority of all the States shall be necessary to a Choice. In every Case, after the Choice of the President, the Person having the greatest Number of remaining Votes of the Electors shall be the Vice President. But if there should remain two or more who have equal Votes, the Senate shall choose from them by Ballot the Vice President.
```

Comment:
I change the process of voting here, from two votes per elector (one from a different state), to one vote per elector, for a candidate from a different state.

This resolves a serious issue with the original text, which has led to the Presidential Ticket system, where each party nominates both a President and a Vice President. The Vice President should not be a position that gets awarded by party officials to someone who owes them a favor. It should be a position that gets awarded to the candidate who was nearly President, but was not as well favored by the Electors as the victor.

It also restricts states from voting for any resident of their own State, which emphasizes the point that the President is a servant of all States, and not merely of their own State.